packer.mk

```diff
@@ -0,0 +1,17 @@
+build-packer:
+	@packer build -var-file=packer/settings.json packer/vhd-image-builder.json
+
+init-packer:
+	@./packer/init-variables
+
+az-login:
+	az login --service-principal -u ${CLIENT_ID} -p ${CLIENT_SECRET} --tenant ${TENANT_ID}
+
+run-packer: az-login
+	@packer version && make init-packer && (make build-packer | tee packer-output)

```

Comment:
When you invoke `make` from inside a Makefile, it should be referred to as `$(MAKE)` according to the [GNU docs](https://www.gnu.org/software/make/manual/html_node/MAKE-Variable.html#MAKE-Variable).

Reply:
fixed